config/schema/rules.schema.yml

```diff
@@ -205,3 +205,38 @@ rules.settings:
     log_level:
       type: integer
       label: 'Log level'
+
+rules.data_type.logger_entry.definition:
```

Comment:
I don't think we need a config schema entry for the logger entry. It is not a config entity, it is just a typed data structure? Does the core Language data type for example have a config schema?